storybook/pages/StatusMembersListPage.qml

```diff
@@ -0,0 +1,141 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import AppLayouts.Chat.panels 1.0
+
+import Storybook 1.0
+
+import utils 1.0
+
+SplitView {
+    id: root
+
+    Logs { id: logs }
+
+    orientation: Qt.Vertical
+
+    property bool globalUtilsReady: false
+    property bool mainModuleReady: false
+
+    // globalUtilsInst mock
+    QtObject {
+        function getEmojiHashAsJson(publicKey) {
+            return JSON.stringify(["👨🏻‍🍼", "🏃🏿‍♂️", "🌇", "🤶🏿", "🏮","🤷🏻‍♂️", "🤦🏻", "📣", "🤎", "👷🏽", "😺", "🥞", "🔃", "🧝🏽‍♂️"])
+        }
+        function getColorId(publicKey) { return colorId.value }
+
+        function getCompressedPk(publicKey) { return "zx3sh" + publicKey }
+
+        function getColorHashAsJson(publicKey) {
+            return JSON.stringify([{colorId: 0, segmentLength: 1},
+                                   {colorId: 19, segmentLength: 2}])
+        }
+
+        function isEnsVerified(publicKey) { return true }
+        function isCompressedPubKey(publicKey) { return true }
+
+        Component.onCompleted: {
+            Utils.globalUtilsInst = this
+            root.globalUtilsReady = true
+
+        }
+        Component.onDestruction: {
+            root.globalUtilsReady = false
+            Utils.globalUtilsInst = {}
+        }
+    }
+
+    // mainModuleInst mock
+    QtObject {
+        function getContactDetailsAsJson(publicKey, getVerificationRequest) {
+            return JSON.stringify({ })
+        }
+        Component.onCompleted: {
+            root.mainModuleReady = true
+            Utils.mainModuleInst = this
+        }
+        Component.onDestruction: {
+            root.mainModuleReady = false
+            Utils.mainModuleInst = {}
+        }
+    }
+    Item {
+        SplitView.fillWidth: true
+        SplitView.fillHeight: true
+        UserListPanel {
```

Comment:
`globalUtilsReady` and `mainModuleReady` are useful only when actual component is instantiated via loader in this way:
```
Loader {
   active: globalUtilsReady && mainModuleReady
   sourceComponent: UserListPanel { ... } 
}
```
Otherwise we can end up with some non-deterministic errors related to initialization order. It may happen that e.g. component in the page will use `Utils.globalUtilsInst` before this `Component.onCompleted` handler is executed:
```
Component.onCompleted: {
    Utils.globalUtilsInst = this
    root.globalUtilsReady = true
}
```

Reply:
removed

Reply:
@alexandraB99 you got me wrong. What I wanted to say is that we need to instantiate `UserListPanel` via loader as demonstrated in the snippet. We want to be sure that those global objects are provided with the mocks before we instantiate `UsrListPanel`. So `globalUtilsReady` and `mainModuleReady` are needed and `Loader` should be used as well :)